C++/test/data_structures/dynamic_array.test.cpp

```diff
@@ -0,0 +1,15 @@
+#define DYNAMIC_ARRAY
+#define CATCH_CONFIG_MAIN
+
+#include "third_party/catch.hpp"
+#include "source/path_to/source_file.cpp"
+
+// Add your test cases here:
+#ifndef DYNAMIC_ARRAY
+
+
```

Comment:
Looks like you forgot to add the tests :)

In case you aren't quite sure about what kind of tests to add or how to add them, you can take a look at the [unit test for DisjointSet](https://github.com/faheel/Algos/blob/master/C%2B%2B/test/data_structures/disjoint_set.test.cpp) for some inspiration.